engines/common/nucleus/particles/logo.yaml

```diff
@@ -20,6 +20,12 @@ form:
       type: input.imagepicker
       label: Image
       description: Select desired logo image.
+      
+    url_disable:
+      type: input.checkbox
+      label: Disable Url
+      description: Disable image url.
```

Comment:
Can you change `Url` to `URL`. Or perhaps even just `No Link` fits better.
Also the description to `Renders Logo/Image without any link`